lib/index.d.ts

```diff
@@ -255,6 +255,11 @@ declare namespace $RefParser {
      * Set this to `true` if you're downloading files from a CORS-enabled server that requires authentication
      */
     withCredentials?: boolean;
+
+    /**
+     * Provide `fetch` function
+     */
+    fetch: typeof globalThis["fetch"];
```

Comment:
Let's add to the top of the file.
We cannot assume the consumer will have the `dom` lib included in their TS config.

```suggestion
    // eslint-disable-next-line
   /// <reference lib="dom" />
```